src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/Utilities/BaseTest.cs

```diff
@@ -444,26 +430,6 @@ protected string GetPathToAapt ()
 			return GetPathToLatestBuildTools (exe);
 		}
 
-		[OneTimeSetUp]
-		public void FixtureSetup ()
-		{
-			// Clean the Resource Cache.
-			if (!TestEnvironment.IsRunningOnCI)
-				return;
-			if (Directory.Exists (CachePath)) {
```

Comment:
This brings up an interesting thought. Should we be deleting this Cache for the user? or issue a warning that they can now remove this directory? It might be taking up valuable space. 

Its probably something for the release notes I think.

Reply:
Yes, we should probably document on the next release notes for 10.2 that will expand upon: https://docs.microsoft.com/en-us/xamarin/android/release-notes/10/10.1#xa0121-deprecation-warning-for-old-xamarinandroidsupport-library-versions

I don't know about automatically deleting, though. You can install VS side-by-side on Windows, and they could have 2017, etc.